api/rds.proto

```diff
@@ -300,7 +300,9 @@ message Route {
     RedirectAction redirect = 3;
   }
 
-  // See base.Metadata description.
+  // See base.Metadata description for message structure. Metadata can
+  // be used to provide the Router filter additional configuration for
```

Comment:
It's also used for stats/logging purposes, not just config.

Reply:
Then perhaps a better sentence would be `The Metadata field can be used to provide the Router filter additional information about the Route. It can be used for configuration, stats, and logging.` 

Also, for my personal understanding. Do you mean that it is currently used that way in the codebase, or it _can_ be used that way. If it is the former, can you point me to where it is being used that way, I can't seem to find an example.

Reply:
I think that wording sounds good. It's actually the latter, but as soon as we have proto-based logging, we're just going to be dumping the entire metadata that gets merged across the request path into the proto log, so that's one place this will be appearing in a non-config context.

Reply:
makes sense, thanks. I thought I was going crazy because I didn't remember/couldnt find where that was the case.